docs/modules/ROOT/pages/list-of-camel-quarkus-extensions.adoc

```diff
@@ -1,10 +1,143 @@
 [list-of-camel-quarkus-extensions]
 = List of Apache Camel extensions for Quarkus
 
-As of Camel Quarkus {camel-quarkus-last-release} the following Camel components are supported on Quarkus:
+As of Camel Quarkus {camel-quarkus-last-release} the following Camel artifacts are supported on Quarkus:
 
-include::{partialsdir}/component-extensions.adoc[]
+== Camel Components
 
-In addition to the above extensions there are the following ancillary extensions:
+// components: START
+Number of Camel components: 22 in 18 JAR artifacts (0 deprecated)
 
-include::{partialsdir}/common-extensions.adoc[]
+[width="100%",cols="4,1,5",options="header"]
+|===
+| Component | Available From | Description
```

Comment:
A non-native speaker's wording notice: Would maybe "Since" be a better alternative for "Available from"? I mean most Java devs should know the `@since` JavaDoc tag that means exactly what we want to express here.